backend/editor/test_api.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+from fastapi.testclient import TestClient
+
+from .api import app
+
+client = TestClient(app)
```

Comment:
you should not use a global variable but a fixture https://docs.pytest.org/en/latest/explanation/fixtures.html#about-fixtures